wrap/python/gtsam_py/tests/test_Cal3Unified.py

```diff
@@ -0,0 +1,44 @@
+import unittest
```

Comment:
The same question for all tests and examples here: 
(1) do we have all tests and examples from cython, or is this a next PR?
(2) I remember adding docstrings and cleaning up some things. Did you compare these new tests with the old tests in cython?
3) do they all run?